ephios/settings.py

```diff
@@ -65,6 +65,7 @@
     "django.contrib.auth.middleware.AuthenticationMiddleware",
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
+    "csp.middleware.CSPMiddleware",
```

Comment:
I'm gonna read some of the documentation for django-csp. 
Is the default configuration everything we want?

Reply:
yes. this restricts everything to self.

Reply:
nice